frontend/src/Components/common/Dashboard.tsx

```
@@ -0,0 +1,392 @@
+                  </button>
+                ))}
+              </div>
+            </div>
+          </div>
+        )
+
+      case "achievements":
+      case "members":
+      case "projects":
+      case "dsa-questions":
+        const currentItem = menuItems.find((item) => item.id === activeSection)
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)